packages/novu/package.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "novu",
-  "version": "2.2.0-rc.2",
+  "version": "2.2.0",
   "description": "Novu CLI. Run Novu Studio and sync workflows with Novu Cloud",
   "main": "src/index.js",
   "publishConfig": {
-    "registry": "https://registry.npmjs.org",
     "access": "public"
   },
   "private": false,
```

Comment:
chore: see `@novu/shared` dependency in `dependencies`, it should be udpated to use `workspace:*` dependency (got accidentally bumped in the last release)

Reply:
Fixed.